etl/steps/data/garden/who/2025-04-07/gho_smoking.meta.yml

```diff
@@ -107,4 +126,77 @@ tables:
+
+          ##### Definition
+          Prices were collected of the most sold brand of cigarettes (pack of 20 sticks).
+
+          Primary collection of price data in this report involved surveying two types of retail outlets from the capital city of each country, defined as follows:
+
+          Supermarket/hypermarket: chain or independent retail outlets with a selling space of over 2500 square metres and a primary focus on selling food/beverages/tobacco and other groceries. Hypermarkets also sell a range of non-grocery merchandise.
+          Kiosk/newsagent/tobacconist/independent food store: small convenience stores, retail outlets selling predominantly food, beverages and tobacco or a combination of these (e.g. kiosk, newsagent or tobacconist) or a wide range of predominantly grocery products (independent food stores or independent small grocers).
+
+          Prices were collected in the local currency in each country. The prices were also converted in international dollars at purchasing power parity (PPP) using the International Monetary Fund (IMF)’s World Economic Outlook implied PPP conversion rates for 2022. Prices were also converted in US$ at official exchange rate using IMF’s exchange rate conversion of national currency per US$, end of period data for July 2022.
 
+
+          ##### Method of measurement
+          Prices provided in different quantities (or volume sizes for e-liquids) were standardized to 20 sticks for cigarettes and Heated Tobacco Products (HTPs).
+
+          Prices in local currency were divided by the IMF’s World Economic Outlook implied PPP conversion rates for 2022 for the conversion in international dollars at purchasing power parity (PPP).
+
+          Prices in local currency were also divided by the IMF’s exchange rate conversion of national currency per US$ end of period data for July 2022 for the conversion in US$ at official exchange rates.
+
+          ##### Limitations
+          In most countries, national level brand-specific price statistics were not available, thus the retail price data were collected from one supermarket/hypermarket or kiosk/newsagent/tobacconist/independent food store usually in the capital city of the country (where survey respondents were mostly located), therefore potentially not being nationally representative.
+      tobacco_ad_ban:
+        title: Bans on tobacco advertising
+        unit: ""
+        short_unit: ""
+        description_short: |-
+          The level of bans on tobacco advertising are defined as follows: 1) no ban, or ban that does not cover national TV, radio and print media advertising; 2) ban on national TV, radio and print media advertising only; 3) ban on national TV, radio and print media, as well as on some, but not all other forms of advertising and 4) complete ban on all forms of advertising.
```

Comment:
Minor, but there are some small differences between this and the subtitle used in the chart (mostly capitalisation), I'd sync them so they are the same.